controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -190,15 +195,15 @@ func (r *ClowdAppReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	// Delete all resources that are not used anymore
 	if !requeue {
-		r.Recorder.Eventf(&app, "Normal", "SuccessfulCreate", "created", app.GetClowdName())
+		r.Recorder.Eventf(&app, "Normal", "SuccessfulCreate", "Clowdapp created [%s]", app.GetClowdName())
```

Comment:
wouldn't this event also fire for all reconciliations after initial creation?

Reply:
Yes - it's really a successful reconciliation message - we could omit entirely